report/Motivation/Motivation.tex

```diff
@@ -0,0 +1,58 @@
+
+In the research paper "The impact of inadequate customer collaboration on self-organizing Agile teams" by Hoda, Noble and Marshall \cite{Hoda2011TheIO} they followed 30 Agile practitioners from 16 different organizations across New Zealand and India over a 3-year period. Their objective was to study "the importance of adequate customer involvement in Agile projects, and the impact of different levels of customer involvement on real-life agile projects.\cite{Hoda2011TheIO}". In their research, they identified a set of reasons for why it has shown difficult to involve customers in an agile project, reasons such as: "skepticism and hype, the distance factor, lack of time commitment, dealing with large customers, fixed-bid contracts, and ineffective customer representatives.\cite{Hoda2011TheIO}". They also identified the impact of lacking customer involvement in a project, such as: "pressure to over-commit, problems in gathering and clarifying requirements, problems in prioritizing requirements, problems in securing feedback, loss of productivity, and in extreme cases, business loss.\cite{Hoda2011TheIO}".
+
+Throughout the research paper by Hoda et al., they present statements from the participants that provide vital insights into the difficulties of customer involvement. On the fact that agile software development requires more customer involvement than traditional software development (read plan-driven), the participants answered: "Commitment for that time from business ... that's something that is not normally there in a [traditional] software development project because [customers] throw [the project] over the wall and do nott have to worry about it for 6 months!" and "Sometimes [customers] only want to come back and see in 6 months what happened [in development]." \cite{Hoda2011TheIO}. From this, we can learn that the customers' understanding of what it means to "order" a new software system is not aligned with the needs of an agile team. Another participant in the research said that "To get the client involved in the process I think is the most difficult part of Agile.\cite{Hoda2011TheIO}". this statement shows that the problem is not a trivial problem. Regarding the lack of time commitment from the customers the participants had statements such as "[customers] can talk about high level 'I want to have Taj Mahal' but of granite or marble? They do not even have time to talk about that!\cite{Hoda2011TheIO}". This tells us that customers have a clear end-goal of what the system should be, in their heads, but they do not have the time to clarify or they do not find it necessary to do. This could be coursed by customers being domain experts and thinking that it is obvious what they need or it could be that they simply do not have the time to sit down and explain it. Other issues regarding requirements identified in the research paper are:
```

Comment:
Kan faktisk godt være at der skal rettes mere på den her. Vi skal have fundet ud af hvordan vi gør med kilden, men kig i det jeg har skrevet før denne comment.
```suggestion
Throughout the research paper by Hoda et al., they present statements from the participants that provide vital insights into the difficulties of customer involvement. 
The fact is that agile software development requires more customer involvement than traditional software development (read plan-driven), the participants answered: \\
"Commitment for that time from business ... that's something that is not normally there in a [traditional] software development project because [customers] throw [the project] over the wall and do not have to worry about it for six months!" and "Sometimes [customers] only want to come back and see in six months what happened [in development]." \cite{Hoda2011TheIO}. \\ 
From this, we can learn that the customers' understanding of what it means to "order" a new software system is not aligned with the needs of an agile team. 
Another participant in the research said that "To get the client involved in the process I think is the most difficult part of Agile."\cite{Hoda2011TheIO}. 
This statement shows that the problem is not a trivial problem. 
Regarding the lack of time commitment from the customers the participants had statements such as "[customers] can talk about high level 'I want to have Taj Mahal' but of granite or marble? They do not even have time to talk about that!"\cite{Hoda2011TheIO}. 
This tells us that the customers have a clear end-goal of what the system should be, in their heads, but they do not have the time to clarify or they do not find it necessary to do. 
This could be caused by customers being domain experts and thinking that it is obvious what they need or it could be that they simply do not have the time to sit down and explain it. 
Other issues regarding requirements identified in the research paper are:
```